packages/firestore/src/local/local_store.ts

```diff
@@ -1305,6 +1329,11 @@ export function applyBundleDocuments(
   const documentBuffer = localStoreImpl.remoteDocuments.newChangeBuffer({
     trackRemovals: true // Make sure document removals show up in `getNewDocumentChanges()`
   });
+
+  const umbrellaTargetData = await allocateTarget(
```

Comment:
Small inline comment would be nice here (a shortened version of what you did above - or you can just inline `umbrellaTarget` and move the comment here).

Reply:
Done.